src/main/java/frc/robot/Robot.java

```diff
@@ -45,7 +48,10 @@ public class Robot extends TimedRobot {
   public void robotInit() {
     // Instantiate our RobotContainer.  This will perform all our button bindings, and use the subsystems needed
     // for the specific robot
-
+    //Draft
+    var talonFXConfigs = new TalonFXConfiguration();
```

Comment:
This is probably the wrong place for this.   Should it be in some subsystem?

Reply:
I agree. Will fix.